templates/web/base/admin/_translations.html

```diff
@@ -0,0 +1,39 @@
+[% IF languages.size > 1 %]
+<h2>Translations</h2>
+<table>
+  <tr>
+    <th>Language</th>
+    <th>Translation</th>
+  </tr>
+  [% FOREACH translation IN translations %]
+  <tr>
+    <td><a href="[% action _ '/translation/' _ translation.lang %]">[% translation.lang %]</a></td>
+    <td>[% translation.msgstr %]</td>
```

Comment:
I realise this probably won't be done often, but it would be nice if translations could be edited from this table, rather than having to go to a separate page?